server/manual-rendering.md

```diff
@@ -0,0 +1,63 @@
+### Manual rendering
```

Comment:
I'm not sure that demonstrating manual rendering is useful to students - it feels like just in case information that they don't need to know and will never act on. The main tool they need to know in my opinion is that res.render('view', object) will use handlebars to render the view.hbs file and that the second param needs to be an object (not an array or something else)